### PR TITLE
elgato.streamdeck version 5.0.0.14247

### PR DESCRIPTION
**Pull request by zephiowl**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/17353)

### CODE REVIEW

File: manifests/e/elgato/streamdeck/5.0.0.14247/elgato.streamdeck.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate
+
+PackageIdentifier: elgato.streamdeck
```
Comment (OfficialEsco) [inline manifests/e/elgato/streamdeck/5.0.0.14247/elgato.streamdeck.installer.yaml:3]:
```suggestion
PackageIdentifier: Elgato.StreamDeck
```

File: manifests/e/elgato/streamdeck/5.0.0.14247/elgato.streamdeck.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Created using wingetcreate
+
+PackageIdentifier: elgato.streamdeck
```
Comment (OfficialEsco) [inline manifests/e/elgato/streamdeck/5.0.0.14247/elgato.streamdeck.locale.en-US.yaml:3]:
```suggestion
PackageIdentifier: Elgato.StreamDeck
```

File: manifests/e/elgato/streamdeck/5.0.0.14247/elgato.streamdeck.yaml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# Created using wingetcreate
+
+PackageIdentifier: elgato.streamdeck
```
Comment (OfficialEsco) [inline manifests/e/elgato/streamdeck/5.0.0.14247/elgato.streamdeck.yaml:3]:
```suggestion
PackageIdentifier: Elgato.StreamDeck
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
The PackageIdentifier is causing issues and have been moved to Elgato.StreamDeck, there is just one PR remaining which will be moved once https://github.com/microsoft/winget-pkgs/pull/16601 is approved..

I would suggest you do `wingetcreate update -i Elgato.StreamDeck -v 5.0.0.14247 -u https://edge.elgato.com/egc/windows/sd/Stream_Deck_5.0.0.14247.msi` instead of fixing this one.

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
`manifests/e/elgato/streamdeck/5.0.0.14247/`
You need to change the path to
`manifests/e/Elgato/StreamDeck/5.0.0.14247/`

**Comment by Xanthus58** [comment]
> `manifests/e/elgato/streamdeck/5.0.0.14247/`
> You need to change the path to
> `manifests/e/Elgato/StreamDeck/5.0.0.14247/`

I've gone ahead and ran through your suggestion here "I would suggest you do wingetcreate update -i Elgato.StreamDeck -v 5.0.0.14247 -u https://edge.elgato.com/egc/windows/sd/Stream_Deck_5.0.0.14247.msi instead of fixing this one."

and resubmitted it for review you can find it here. 
https://github.com/microsoft/winget-pkgs/pull/17416

**Comment by OfficialEsco** [comment]
Good good, you can close this one then :)

**Pull request closed without merging**